testing/src/main/scala/org/typelevel/ci/testing/arbitraries.scala

```diff
@@ -33,7 +33,13 @@ object arbitraries {
     // Also focus on characters that are cased at all
     val lowers = chars.filter(_.isLower)
     val uppers = chars.filter(_.isUpper)
-    val genChar = Gen.oneOf(weirdCharFolds, weirdStringFolds, lowers, uppers, arbitrary[Char])
+    val genChar = Gen.oneOf(
+      Gen.oneOf(weirdCharFolds),
+      Gen.oneOf(weirdStringFolds),
+      Gen.oneOf(lowers),
+      Gen.oneOf(uppers),
+      arbitrary[Char]
+    )
```

Comment:
New compiler flags found a bug here. Instead of a `Gen[Char]` we had a `Gen[Object]` where some of the objects were `Seq[Char]`s.